debian/tests/testsuite

```diff
@@ -0,0 +1,17 @@
+#!/bin/sh
+set -efu
+set -x
+
+pys="$(py3versions -s 2> /dev/null)"
+
+cp -rva tests "$AUTOPKGTEST_TMP/"
+cd "$AUTOPKGTEST_TMP/tests"
+
+# Resolve "from tests import utils"
+PYTHONPATH=".."
```

Comment:
the purpose of autopkgtest is tests the packages as is installed. So, please remove PYTHONPATH.